### PR TITLE
Add support for HTTPS connection (fixes #29)

### LINKED ISSUE #29: Add support for HTTPS connection
The library should allow to connect to the  HTTPS endpoint.

### PR DESCRIPTION
**Pull request by thotypous**
Just a quick and dirty patch. Please let me know if you have any suggestions.

### CODE REVIEW

**Review by zmedico — APPROVED** [review]
Looks good. I'm thinking that it should be possible to auto-negotiate the scheme if it's not specified, but that would add some complexity.

**Comment by otoolep** [comment]
Thanks for your contribution.

File: src/pyrqlite/connections.py [hunk 1]
```diff
@@ -40,11 +40,12 @@ class Connection(object):
         NotSupportedError,
     )
 
-    def __init__(self, host='localhost', port=4001,
+    def __init__(self, scheme='http', host='localhost', port=4001,
```
Comment (otoolep) [inline src/pyrqlite/connections.py:43]: With this change, perhaps you need the option to allow trusting of unsigned, or self-signed certs.

Comment (zmedico) [reply src/pyrqlite/connections.py:43]: And it would be interesting to have the unit tests generate some certs and use them.

**Pull request merged**